stdlib/public/Platform/Platform.swift

```diff
@@ -187,6 +209,7 @@ public func open(
   return _swift_Platform_open(path, oflag, 0)
 }
 
+#if !os(Windows) || CYGWIN
 public func open(
   _ path: UnsafePointer<CChar>,
   _ oflag: Int32,
```

Comment:
You've shimmed `open()` with the mode on Windows, why did you have to hide this overload?